packages/@coorpacademy-components/src/template/skill-detail/index.js

```diff
@@ -0,0 +1,241 @@
+            : primarySkinColor
+        }}
+        icon={{
+          position: 'left',
+          faIcon: {
+            name: 'play',
+            color: '#FFFFFF',
+            size: 16
+          }
+        }}
+        onClick={onClick}
+      />
+    </div>
+  );
+};
+
+ContinueLearningButton.contextTypes = {
+  skin: Provider.childContextTypes.skin,
+  translate: Provider.childContextTypes.translate
+};
+
+ContinueLearningButton.propTypes = {
+  onClick: PropTypes.func
+};
+
+const SkillDetail = (props, context) => {
+  const {
+    title,
+    description,
+    metrics,
+    focused,
+    availableForReview,
+    ongoingCourses,
+    skillIncludedCourses,
+    filters,
+    sorting,
+    onBackClick,
+    onReviewClick,
+    onContinueLearningClick
+  } = props;
+  const {score, content, questionsToReview, contentCompleted = 0} = metrics;
+  const {translate} = context;
+
+  const [showMore, setShowMore] = useState(false);
+
+  const handleShowMore = useCallback(() => setShowMore(!showMore), [setShowMore, showMore]);
+
+  const Description = useCallback(() => {
+    return (
+      <div className={classnames(style.description, !showMore && style.truncate)}>
+        {description}
+      </div>
+    );
+  });
+
+  const ProgressBar = useCallback(() => {
+    if (score === undefined) return null;
+    const progressBarColor = '#3EC483';
+    const inlineProgressValueStyle = {
+      backgroundColor: progressBarColor,
+      width: `${score}%`
+    };
+
+    return (
+      <div className={style.progressBarWrapper}>
+        <div
+          data-name="progress"
+          className={style.progress}
+          style={inlineProgressValueStyle}
+          role="progressbar"
+        />
+      </div>
+    );
+  }, [score]);
+
+  return (
+    <div className={style.backgroundContainer}>
+      <div className={style.container}>
+        <div className={style.backButtonWrapper} onClick={onBackClick}>
+          <Icon
+            iconName="arrow-left"
+            size={{
+              faSize: 14,
+              wrapperSize: 14
+            }}
+          />
+        </div>
+        <div className={style.ctaContainer}>
+          <div>
+            {focused ? (
+              <div className={style.skillFocusBadge}>
+                <Icon
+                  iconName="bullseye-arrow"
+                  backgroundColor="#DDD1FF"
+                  size={{
+                    faSize: 10,
+                    wrapperSize: 16
+                  }}
+                />
+                {translate('skill_focus')}
+              </div>
+            ) : null}
+            <div className={style.title}>{title}</div>
+            {description ? (
+              <>
+                <Description />
+                {description.length >= 226 ? (
+                  <div className={style.showMoreWrapper} onClick={handleShowMore}>
+                    {showMore ? translate('Show less') : translate('Show more')}
```

Comment:
Show less feels a bit to proeminent/visible IMO, cc @wernerdurand 
![Capture d’écran 2024-07-22 à 15 00 03](https://github.com/user-attachments/assets/3d27675a-054e-42da-8310-942b4f5dea8d)